src/hera/workflows/_mixins.py

```diff
@@ -680,12 +680,8 @@ def _get_arguments(self, **kwargs) -> List:
         # these are the already set parameters. If a user has already set a parameter argument, then Hera
         # uses the user-provided value rather than the inferred value
         kwargs_arguments = kwargs.get("arguments", [])
-        kwargs_arguments = (
-            kwargs_arguments if isinstance(kwargs_arguments, List) else [kwargs_arguments]
-        )  # type: ignore
-        arguments = (
-            self.arguments if isinstance(self.arguments, List) else [self.arguments] + kwargs_arguments
-        )  # type: ignore
+        kwargs_arguments = kwargs_arguments if isinstance(kwargs_arguments, List) else [kwargs_arguments]  # type: ignore
```

Comment:
The long line here is because ruff's `line-length` isn't a "max" enforced length https://docs.astral.sh/ruff/settings/#line-length